### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ch1bo**
Reviewed and suggested changes of paths used. Of course those would need to be fixed in the code, but could not think of a better way to provide feedback than a PR in itself.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -140,17 +140,17 @@ To connect with another player, one needs to configure a file named  `peers.json
 
 This file should be placed in a directory named after the cardano network used under one's home directory:
 
-* `hychess --network Preview` :point_right: `~/.config/hydra-node/preview/`
-* `hychess --network Preprod` :point_right: `~/.config/hydra-node/preprod/`
+* `hychess --network Preview` :point_right: `~/.config/hydra-chess/preview/`
+* `hychess --network Preprod` :point_right: `~/.config/hydra-chess/preprod/`
```
Comment (ch1bo) [inline README.md:144]: Would it be possible to keep the config and cache in the xdg dirs for `hydra-chess` instead?

i.e. use the project name as a qualifier to not conflict with any potential forks of this easily

BTW: Love the usage of XDG directories :heart:

Comment (ghost) [reply README.md:144]: Hmm, yes I see your point. I was afraid of mixing up a whole lot of different configuration, executables, data files in a single directory without much control and the possibility of conflicts _there_. How about a top-level directory `hydra-chess` with subdirectories for components, eg. :
```
~/.config/hydra-chess/preview/hydra-node/xxx
~/.config/hydra-chess/preview/cardano-node/xxx
...
```
?

**Comment by ghost** [comment]
Thanks a lot for the review!

**Comment by ghost** [comment]
Also, don't look too much at the code, or you'll run away screaming 😂 
This is really a prototype and I messed up the design, would be happy to go to the drawing board together for v0.2

**Comment by ch1bo** [comment]
Close din favor of tracking this as an issue: https://github.com/cardano-scaling/hydra-chess/issues/2

**Pull request closed without merging**